app/main/appium-method-handler.js

```diff
@@ -80,7 +81,14 @@ export default class AppiumMethodHandler {
 
   async executeMethod (methodName, args = []) {
     let res = {};
-    if (methodName !== 'source' && methodName !== 'screenshot') {
+
+    // Specially handle the tap method
+    if (methodName === 'tap') {
+      res = await (new wd.TouchAction(this.driver)).tap({x: args[0], y: args[1]}).perform();
+    } else if (methodName === 'swipe') {
+      const [startX, startY, endX, endY]  = args;
+      res = await (new wd.TouchAction(this.driver)).press({x: startX, y: startY}).moveTo({x: endX, y: endY}).release().perform();
```

Comment:
i think this would look cleaner separating out the individual commands on separate lines:

```js
res = await (new wd.TouchAction(this.driver))
  .press({x: startX, y: starty})
  .moveTo({x: endX, y: endY})
  .release()
  .perform();
```